schema-render.js.org

Barrior opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But it looks like some assets are not getting loaded because of changed paths. If you could fix this so I can have a look at your page and merge afterwards.

Barrior commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But it looks like some assets are not getting loaded because of changed paths. If you could fix this so I can have a look at your page and merge afterwards.

I fixed this problem, sorry for the inconvenience.

indus commented:
Welcome https://schema-render.js.org
